study.md

```diff
@@ -85,7 +88,7 @@ Using Hash methods and string interpolation in Ruby, create a string using
 conspiracy theories.".
 
 ```ruby
-# your answer here
+"My name is #{dale[:given_name]} #{dale[:family_name]} and I'm an #{dale[:occupation]} that enjoys #{dale[:hobbies(1)]}."
```

Comment:
`dale[:hobbies(1)]` should be `dale[:hobbies[1]]`.  The way that it's currently written will attempt to execute the symbol `:hobbies` as if it were a function.